### PR TITLE
Fix the norm of complex numbers in clipping

### LINKED ISSUE #144: Norm of complex numbers
When using `clip_by_global_norm` on gradients of complex parameters, it seems we need to change `jnp.square(x)` to `x.conj() * x` in the function `global_norm` in `_src/linear_algebra.py`.

How is the current status of complex number support in Optax? I'm using neural networks in quantum physics, and I'd be happy to help JAX community to enhance complex number support. I guess I'll encounter more problems about it and I'll report them then.

### PR DESCRIPTION
**Pull request by wdphy16**
Fixes #144

### CODE REVIEW

**Comment by PhilipVinc** [comment]
Bump. can this be merged?

**Comment by PhilipVinc** [comment]
Bump @mkunesch @rosshemsley , can this be reviewed and eventually merged?

**Comment by wdphy16** [comment]
Bump @mkunesch @rosshemsley @fabioviola , any update on this?

**Comment by PhilipVinc** [comment]
Bump again. I'm going to try my luck with @hbq1 .
I'm sorry we are so insistent but we'd love this merged. 
Or if optax is more of an internal deepmind project that is not open for external contributions we'd love to know...

**Comment by wdphy16** [comment]
Closed in favor of #279
